pkg/uwb-dw1000/Makefile.dep

```diff
@@ -0,0 +1,14 @@
+USEPKG += uwb-core
```

Comment:
This dependency seems wrong to me. Isn't `uwb-core` that require `uwb-dw1000` ?

Reply:
Its not a clear line, `uwb-dw1000` needs the `dpl` part of `uwb-core`. I'll see i I can reflect that dependency better

Reply:
For now I'll remove the dependency since `uwb-dw1000` is not yet usable stand alone